.github/resources/xcm-weight-template.hbs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020-2022 Manta Network.
```

Comment:
Where can I find xcm weight template?

Reply:
It's in the .github/resource folder like the other one.

Reply:
Sorry, I didn't express it clearly. I mean where the template is from.

Reply:
It's from the Cumulus repo https://github.com/paritytech/cumulus/blob/master/templates/xcm-bench-template.hbs